src/m365/teams/commands/channel/channel-get.ts

```diff
@@ -44,7 +44,7 @@ class TeamsChannelGetCommand extends GraphCommand {
     }
 
     const requestOptions: any = {
-      url: `${this.resource}/beta/groups?$filter=resourceProvisioningOptions/Any(x:x eq 'Team') and displayName eq '${encodeURIComponent(args.options.teamName as string)}'`,
+      url: `${this.resource}/v1.0/groups?$filter=displayName eq '${encodeURIComponent(args.options.teamName as string)}'`,
```

Comment:
In the discussion on #2505 we decided to implement the filter for Teams-groups in the code. It seems like we're missing this here. Have you omitted it on purpose or by accident?

Reply:
I missed it. Will update the code and submit the Pull request.

Reply:
Awesome! Thank you!